app/components/ApiEndpoint.vue

```diff
@@ -120,28 +127,57 @@ const responseBodyExample = computed(() => {
 				<ApiParams :param="requestBodySchema" />
 			</div>
 
-			<div
-				v-for="responseBodyObject, code, index in responseBodyObjects"
-				:key="code"
-				class="mb-12 last:mb-0"
-			>
-				<ProseH4 :id="slugify(operation.summary!) + '-' + responseBodyObject.code + '-response'">
-					{{code}} Response
+			<div class="mb-12 last:mb-0">
+				<ProseH4 class="mt-12">
+					Responses
 				</ProseH4>
-				<ProseP v-if="responseBodyObject.description">
-					{{ responseBodyObject.description }}
-				</ProseP>
-				<ApiParams v-if="flattenedResponseBodySchemas[index]" :param="flattenedResponseBodySchemas[index]" />
+				<UTabs
+					variant="link"
+					:items="Object.keys(responseBodyObjects).map((code, index) => ({
+						label: code,
+						meta: responseBodyObjects[code],
+						index,
+					}))"
+					:unmount-on-hide="false"
+				>
+					<template #default="{ item }">
+						<UBadge
+							variant="soft"
+							size="lg"
+							class="font-mono"
+							color="neutral"
+						>
+							<UChip
+								standalone
+								inset
+								:color="item.label.toString().startsWith('2') ? 'success' : 'error'"
+							/>
+							{{ item.label }}
```

Comment:
I wonder if instead of just `200`, if it could say like `200 Ok` or `403 Forbidden` so it is more informative!